web/src/lib/components/asset-viewer/detail-panel-rating.svelte

```diff
@@ -0,0 +1,102 @@
+<script lang="ts">
+  import { handleError } from '$lib/utils/handle-error';
+  import { updateAsset, type AssetResponseDto } from '@immich/sdk';
+  import { t } from 'svelte-i18n';
+
+  export let asset: AssetResponseDto;
+  export let isOwner: boolean;
+
+  const countStars = 5;
+
+  $: rating = asset.exifInfo?.rating || 0;
+  $: currentRating = rating;
```

Comment:
Why are `currentRating` and `rating` reactive?

Reply:
You're right, it only needs one reactive variable. Has been corrected.